### PR TITLE
pcre 8.43

### PR DESCRIPTION
**Pull request by lembacon**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by mistydemeo** [comment]
I have #37281 open to fix swig, which broke for unrelated reasons.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

File: Formula/pcre.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Pcre < Formula
   desc "Perl compatible regular expressions library"
   homepage "https://www.pcre.org/"
-  url "https://ftp.pcre.org/pub/pcre/pcre-8.42.tar.bz2"
-  mirror "https://www.mirrorservice.org/sites/ftp.exim.org/pub/pcre/pcre-8.42.tar.bz2"
```
Comment (leonklingele) [inline Formula/pcre.rb]: Why has the mirror been removed?

Comment (mistydemeo) [reply Formula/pcre.rb]: I'll add it back when it gets pulled.

**Comment by lembacon** [comment]
Merged, and also added the `mirror` back. Thanks @mistydemeo for helping with the failing `swig` test and also thanks @leonklingele for pointing the missing of the `mirror` out!

**Comment by tstack** [comment]
There's a bug in 8.43 that causes crashes for programs that fork() (see https://bugs.exim.org/show_bug.cgi?id=2334).  Is there a chance this upgrade could be rolled back?

**Pull request closed without merging**